exec/jdbc-all/pom.xml

```diff
@@ -329,8 +386,6 @@
             </includes>
             <excludes>
               <exclude>io.protostuff:*</exclude>
-              <exclude>${calcite.groupId}:calcite-core</exclude>
```

Comment:
It is better to have exclusions here since they are applied to all dependencies.

Reply:
When they're made conventional Maven exclusions then they act transitively resulting in size savings. They also make the mvn depedency:tree tool more useful.